sdk/python/src/dagger/telemetry.py

```diff
@@ -64,19 +66,9 @@ def otel_configured() -> bool:
     return any(k for k in os.environ if k.startswith("OTEL_"))
 
 
-def otel_enabled(name: str = "") -> bool:
-    """Checks if a specific OpenTelemetry instrumentation is disabled.
-
-    Based on environment variables:
-    - If name is not provided, checks for global OTEL_SDK_DISABLED=true.
-    - Otherwise, checks for OTEL_PYTHON_{name}_INSTRUMENTATION_DISABLED=true.
-    """
-    name = (
-        f"OTEL_PYTHON_{name.upper()}_INSTRUMENTATION_DISABLED"
```

Comment:
There's a `OTEL_PYTHON_LOGGING_INSTRUMENTATION_DISABLED` in the `opentelemetry.sdk` package because logs support is experimental, but this looks for:
```env
OTEL_PYTHON_TRACES_INSTRUMENTATION_DISABLED=true
OTEL_PYTHON_LOGS_INSTRUMENTATION_DISABLED=true
OTEL_PYTHON_METRICS_INSTRUMENTATION_DISABLED=true
```
The `_LOGGING_` (instead of `_LOGS_`) variation is still picked up by the `opentelemetry.sdk` so it's not a problem that they don't match. The idea was to allow the user to disable only one kind of instrumentation rather than the whole thing but I've wanted to simplify this for a while now.